web/client/components/mapcontrols/searchcoordinates/CoordinatesSearch.js

```diff
@@ -194,7 +249,8 @@ CoordinatesSearch.propTypes = {
     onClearCoordinatesSearch: PropTypes.func,
     onZoomToPoint: PropTypes.func,
     onChangeCoord: PropTypes.func,
-    defaultZoomLevel: PropTypes.number
+    defaultZoomLevel: PropTypes.number,
```

Comment:
This seems to be missing in
https://github.com/geosolutions-it/MapStore2/pull/10220/files#diff-455b07539c76115f6dce6c982a88461c05ae2ab7cb8a2a3fac5a2a6ad58ba9cc